src/App.jsx

```diff
@@ -14,45 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
+
+function getPrerapeGoods(goods, sortField, reverseField) {
+  let prepareGood = [...goods];
+
+  prepareGood = prepareGood.sort((good1, good2) => {
+    switch (sortField) {
+      case SORT_FIELD_ALPHABETICALLY:
+        return good1.localeCompare(good2);
+      case SORT_FIELD_LENGTH:
+        return good1[sortField] - good2[sortField];
+      default: return 0;
+    }
+  });
```

Comment:
```suggestion
  prepareGood.sort((good1, good2) => {
    switch (sortField) {
      case SORT_FIELD_ALPHABETICALLY:
        return good1.localeCompare(good2);
      case SORT_FIELD_LENGTH:
        return good1[sortField] - good2[sortField];
      default: return 0;
    }
  });
```


sort and reverse are mutating methods